src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+enum Shapes {
+  circle = 'circle',
+  triangle = 'triangle',
+  rectangle = 'rectangle',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please update the `Shapes` enum to use capital letters for the enum constants as per the good example in the checklist.